src/models/User/createUser/index.js

```diff
@@ -43,9 +42,11 @@ const createUser = async ({
     yahoo_remove_subaddress: false,
     icloud_remove_subaddress: false
   })
-  const existingUsersWithEmail = await database('users')
-    .where({ email })
+
+  const existingUsersWithEmail = await User.query()
+    .where('email', '=', email)
```

Comment:
Shouldn’t you use `.findOne()` here or use `.where({ email })`? Let’s avoid those raw-ish queries when we can.